src/components/projects/view/danger-confirm-dialog.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Button from 'ui/button.svelte';
+  import Dialog from 'ui/dialog.svelte';
+  import Modal from 'ui/modal.svelte';
+
+  export let isOpen = false;
+  export let textYes;
+  export let eventDetail = null;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<Modal bind:isOpen>
+  <Dialog
+    isDanger
+    title="Warning"
+    closeCallback={() => isOpen = false}
+  >
+    <svg slot="icon" src="images/icons/alert-triangle.svg" class="icon" />
+    <p slot="content">
+      <slot />
+    </p>
+    <div slot="content" class="actions">
+      <Button on:click={() => { isOpen = false; dispatch('reject', eventDetail); }}>
+        no, wait
+      </Button>
+      <Button isDanger isFilled on:click={() => dispatch('confirm', eventDetail)}>
+        {textYes}
```

Comment:
Why is this text a parameter (with no default) when the "no text" is not?

Reply:
Not striving for 100% reusability here, this is a page-specific component. As for the default, same thing, there's always a value in our use cases